data-non-standard.js

```diff
@@ -1218,7 +1212,7 @@ exports.tests = [
     firefox12: false,
     firefox16: true,
     firefox17: false,
-    firefox20: true,
+    firefox20: false,
     firefox28: false,
```

Comment:
What I'm trying to say is that in your branch
```
firefox20: false,
firefox28: false,
```

is the same as
```
firefox20: false,
```

because there is no data under key `firefox28` in browsers list, it is represented under `firefox20` in your branch. So `firefox28: false,` is useless and should be deleted as well as other `firefox28` mentions in this file.

By the way I  hope this PR will not be merged. Those browsers are  **very** obsolete and they don't get rendered in table anyway. The data about particular browsers may still be useful for example for cases described in https://github.com/kangax/compat-table/issues/785